x-pack/plugins/uptime/public/pages/monitor.tsx

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { EuiSpacer } from '@elastic/eui';
```

Comment:
I'd expect a test to fail from removing this much markup. Do we have unit tests for this component?